.vscode/launch.json

```diff
@@ -160,7 +160,8 @@
             },
             "presentation": {
                 "group": "main"
-            }
+            },
+            "envFile": "${workspaceFolder}/.env"
```

Comment:
Small thing but helps so i can run plugin server in `local-cdp` mode without needing to modify launch.json